### PR TITLE
ponominalu.ru

### PR DESCRIPTION
**Pull request by galeksandrp**
Issue author: @galeksandrp

### CODE REVIEW

File: src/chrome/content/rules/ponominalu.ru.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ns2.ponominalu.ru different content
+omsk.ponominalu.ru different content
+rnd.ponominalu.ru different content
+sochi.ponominalu.ru different content
+spb.ponominalu.ru different content
+tomsk.ponominalu.ru different content
+tver.ponominalu.ru different content
+vdm.ponominalu.ru different content
+volg.ponominalu.ru different content
+yola.ponominalu.ru different content
```
Comment (J0WI) [inline src/chrome/content/rules/ponominalu.ru.xml]: Could you double check the 'different content' issues?
I think they are working now.

Comment (J0WI) [inline src/chrome/content/rules/ponominalu.ru.xml]: These hosts are redirected server side to https. What content is different?

Comment (galeksandrp) [inline src/chrome/content/rules/ponominalu.ru.xml]: That is due
```bash
INFO Big distance 0.1022: http://ulsk.ponominalu.ru/ (11295) -> https://ulsk.ponominalu.ru/ (11295). Rulefile: src/chrome/content/rules/ponominalu.ru.xml =====
```
and so on. [This is appears even on domains not noted as different content](https://travis-ci.org/EFForg/https-everywhere/jobs/203939740).

Comment (J0WI) [inline src/chrome/content/rules/ponominalu.ru.xml]: I don't really understand the issue. What exactly are you changing in 4778f15?

**Comment by galeksandrp** [comment]
@Hainish https://github.com/EFForg/https-everywhere/blob/e04d0244382535f07f13cd5fba7e7e49799dac03/test/rules/src/https_everywhere_checker/check_rules.py#L160 should not be emitted when 301/302 sent by http page.

**Pull request merged**